contrib/utilities/indent

```diff
@@ -21,6 +21,20 @@
 ##
 ## ---------------------------------------------------------------------
 
+find_astyle() {
+  IFS=':' read -r -a PATHS <<< "$PATH"
```

Comment:
I am worried that this might not work on everybody machine (shell, OSX, etc.). Can we use this logic only if the default ``astyle`` doesn't work?

Reply:
I think the `#!/bin/bash` should make this compatible, but I added a fallback in https://github.com/geodynamics/aspect/pull/3699/commits/1fbe28dc90afdba449819299668b72347f87370f that should return the output of `which astyle` in case the other parts of this function fail.